Dockerfile

```diff
@@ -147,10 +148,13 @@ RUN rm -rf /usr/bin/python && ln /usr/bin/python3.7 /usr/bin/python
 
 RUN curl --silent https://bootstrap.pypa.io/get-pip.py | python
 
-# Set up ubuntu user
+# Set up ubuntu user and home directory with access to users in the root group (0)
 
-RUN groupadd --gid 1000 ubuntu \
-  && useradd --uid 1000 --gid ubuntu --shell /bin/bash --create-home ubuntu
+ENV HOME=/home/ubuntu
+RUN groupadd --gid 1000 ubuntu && \
+  useradd --uid 1000 --gid ubuntu --groups 0 --shell /bin/bash --home-dir ${HOME} --create-home ubuntu && \
+  chown -R ubuntu:0 ${HOME} && \
+  chmod -R g=u ${HOME}
```

Comment:
This can be removed, because it will be done again on line 225-226 below.

Reply:
No problem